### DISCUSSION TITLE
My graduation completion in 2022 but it's saying you're previously graduated

### DISCUSSION THREAD

**Discussion by nameisazeem** [community/community#17766] [GitHub Education]
I created a pull request for GitHub graduation 2022 but it was showing you're previously graduated. My official graduation year is 2022. What can I do to get into this year's graduation ceremony?

**Comment by juanpflores** [comment]
Hey @nameisazeem 👋🏼 
Thanks for letting us know about it. I have reviewed the pull request and got you all set for the event. 👾 Congratulations on graduating this year

**Comment by elisemoe** [comment]
For others having the same problem, [please add a link to your PR in this Issue](https://github.com/education/GitHubGraduation-2022/issues/4290) and we'll manually review your Pull Request.
